app/src/main/java/vn/mbm/phimp/me/editor/editimage/EditImageActivity.java

```diff
@@ -469,6 +459,8 @@ public void onBackPressed() {
             case MODE_PAINT:
                 paintFragment.backToMain();
                 return;
+            default:
+                finish();
```

Comment:
this implementation causes the editor to close directly without any confirmation dialog(which was present already) even if the editing is in intermediate stage.
Please check that.